<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastro_animal</class>
 <widget class="QMainWindow" name="cadastro_animal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pet Shop Management</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>450</y>
      <width>164</width>
      <height>32</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>1934</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_nome_cientifico">
         <property name="text">
          <string>Nome Cientifico:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_nome_cientifico_txt"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_nome_de_bastismo">
         <property name="text">
          <string>Nome de Batismo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_nome_de_bastismo_txt"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_dieta">
         <property name="text">
          <string>Dieta:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_dieta_txt"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_classe">
           <property name="text">
            <string>Classe:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_classe_txt_combo_box">
           <item>
            <property name="text">
             <string>--Selecione--</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Anfíbio</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mamifero</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Reptil</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ave</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_sexo">
           <property name="text">
            <string>Sexo:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_sexo_txt_combo_box">
           <item>
            <property name="text">
             <string>--Selecione--</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>M</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>F</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_tamanho">
           <property name="text">
            <string>Tamanho:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="m_tamanho_txt_spin_box"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_tratador">
           <property name="text">
            <string>Tratador:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="m_tratador_txt_spin_box">
           <property name="maximum">
            <number>9999999</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::DefaultStepType</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_veterinario">
           <property name="text">
            <string>Veterinario:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="m_veterinario_txt_spin_box">
           <property name="maximum">
            <number>9999999</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::DefaultStepType</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_tipo">
             <property name="text">
              <string>Tipo:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="m_tipo_txt_combo_box">
             <item>
              <property name="text">
               <string>--Selecione--</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Nativo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Exótico</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="mamifero_layout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_venenoso">
             <property name="text">
              <string>Venenoso:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="m_venenoso_txt">
             <item>
              <property name="text">
               <string>--Selecione--</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Sim</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Não</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_tipo_veneno">
           <property name="text">
            <string>Tipo de veneno:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_tipo_veneno_txt"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="anfibio_layout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QLabel" name="label_total_de_mudas">
               <property name="text">
                <string>Total de Mudas:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="m_total_de_mudas_spin_box">
               <property name="maximum">
                <number>100000000</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <widget class="QLabel" name="label_ultima_muda">
               <property name="text">
                <string>Última muda:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="m_data_ultima_muda_txt_data_box">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_cor_do_pelo">
         <property name="text">
          <string>Cor do Pelo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_cor_do_pelo_txt"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="ave_layout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_tamanho_bico">
             <property name="text">
              <string>Tamanho do Bico:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="m_tamanho_do_bico_txt_spin_box"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_envergadura_das_asas">
             <property name="text">
              <string>Envergadura das asas:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="m_envergadura_das_asas_txt_spin_box"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_23">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <widget class="QLabel" name="label_uf_origem">
           <property name="text">
            <string>UF de Origem:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_uf_origem"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_22">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <widget class="QLabel" name="label_pais_de_origem">
             <property name="text">
              <string>País de Origem:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="m_pais_origem"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_21">
           <item>
            <widget class="QLabel" name="label_cidade_de_origem">
             <property name="text">
              <string>Cidade de Origem</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="m_cidade_origem"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_24">
       <item>
        <widget class="QLabel" name="label_autorizacao_ibama">
         <property name="text">
          <string>Autorização do IBAMA:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_autorizacao_ibama"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>152</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QLabel" name="label_id_animal">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m_id_spin">
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
